xbmc/Application.cpp

```diff
@@ -2829,7 +2829,7 @@ bool CApplication::PlayFile(CFileItem item, const std::string& player, bool bRes
         options.state = item.GetProperty("savedplayerstate").asString();
         item.ClearProperty("savedplayerstate");
       }
-      else if (item.m_lStartOffset == STARTOFFSET_RESUME)
+      else if (item.m_lStartOffset == STARTOFFSET_RESUME && !(item.HasProperty("StartPercent") && options.startpercent == 0.0f))
```

Comment:
Looking at this again, I think your code would be fine if you just do `else if (item.m_lStartOffset == STARTOFFSET_RESUME && options.startpercent > 0.0f)` which is much clearer I think.

Reply:
Ok, I will re-compile a new version and see if that works. I really wanted to be specific to the StartPercent property and not mess existing behavior otherwise :).